gravitee-apim-console-webui/src/services-ngx/notification-settings.service.ts

```diff
@@ -33,4 +35,12 @@ export class NotificationSettingsService {
   delete(apiId: string, id: string): Observable<NotificationSettings[]> {
     return this.http.delete<NotificationSettings[]>(`${this.constants.env.baseURL}/apis/${apiId}/notificationsettings/${id}`);
   }
+
+  getNotifiers(apiId: string): Observable<Notifier[]> {
+    return this.http.get<Notifier[]>(`${this.constants.env.baseURL}/apis/${apiId}/notifiers`);
+  }
+
+  create(apiId, notificationConfig: NewNotification): Observable<NewNotification[]> {
```

Comment:
Are we sure it returns a `NewNotification[]` 🤔  the `[]` is weird no?

Reply:
Fixed